keras/backend/__init__.py

```diff
@@ -83,7 +81,11 @@
     sys.stderr.write('Using TensorFlow backend.\n')
     from .tensorflow_backend import *
 else:
-    raise ValueError('Unknown backend: ' + str(_BACKEND))
+    try:
+        exec('from ' + _BACKEND + ' import *')
```

Comment:
Using `exec` on arbitrary code would be extremely unsafe.

Here is one safer solution:

```python
backend_module = importlib.import_module(_BACKEND)
globals().update(backend_module.__dict__)
```

However it has one issue: it potentially pollutes the namespace of `keras.backend` with irrelevant entries (such as imports made by the module) and could override some important names such as `epsilon`, etc.

But overall the behavior is the same as with the other backends, so it's probably ok.